docs/Factory.md

```diff
@@ -0,0 +1,103 @@
+<h1 align="center">Fastify</h1>
+
+<a name="factory"></a>
+## Factory
+
+The Fastify module exports a factory function that is used to create new
+<a href="https://github.com/fastify/fastify/blob/master/docs/Server-Methods.md"><code><b>Fastify server</b></code></a>
+instances. This factory function accepts an options object which is used to
+customize the resulting instance. This document describes the properties
+available in that options object.
+
+<a name="factory-http2"></a>
+### `http2` (Status: experimental)
+
+If `true` Node.js core's [HTTP/2](https://nodejs.org/dist/latest-v8.x/docs/api/http2.html)
+HTTP/2 module is used for binding the socket.
+
++ Default: `false`
+
+<a name="factory-https"></a>
+### `https`
+
+An object used to configure the server's listening socket for TLS. The options
+are the same as the Node.js core
+[`createServer` method](https://nodejs.org/dist/latest-v8.x/docs/api/https.html#https_https_createserver_options_requestlistener).
+When this property is `null`, the socket will not be configured for TLS.
+
+This option also applies when the
+<a href="https://github.com/fastify/fastify/blob/master/docs/Factory.md#factory-http2">
+<code><b>http2</b></code>
+</a> option is set.
+
++ Default: `null`
+
+<a name="factory-ignore-slash"></a>
+### `ignoreTrailingSlash`
+
+Fastify uses [find-my-way](https://github.com/delvedor/find-my-way) to handle
+routing. This option may be set to `true` to ignore trailing slashes in routes.
+This option applies to *all* route registrations for the resulting server
+instance.
+
++ Default: `false`
+
+#### Example
+
+```js
+const fastify = require('fastify')({
+  ignoreTrailingSlash: true
+})
+
+// registers both "/foo" and "/foo/"
+fastify.get('/foo/', function (req, reply) {
+  res.send('foo')
+})
+
+// registers both "/bar" and "/bar/"
+fastify.get('/bar', function (req, reply) {
+  res.send('bar')
+})
+```
+
+<a name="factory-json-limit"></a>
+### `jsonBodyLimit`
+
+Defines the maximum payload, in bytes, the server is allowed to accept for
+request bodies when using the internal `application/json` content type parser.
+
++ Default: `1048576` (1MiB)
+
+<a name="factory-logger"></a>
+### `logger`
+
+Fastify includes built-in logging via the [Pino](https://getpino.io/) logger.
+This property is used to configure the internal logger instance.
+
+The possible values this property may have are:
+
++ Default: `false`. The logger is disabled. All logging methods will point to a
```

Comment:
I keep the default value as last statement like other sections

Reply:
I don't agree on this one. It's a complicated option and it's better to know upfront what the initial value is. It's the "last statement" on the others because there isn't anything else to write after the opening description. This one is actually in the same position: first item in the list of possibilities after the opening description.